action.yml

```diff
@@ -13,21 +13,37 @@ inputs:
   text:
     description: "The message text to post"
     required: true
+  unfurl_links:
+    description: "Whether links in the message should be unfurled"
+    type: boolean
+    default: true
 outputs:
   ts:
     description: "The timestamp ID of the message that was just posted"
+    value: ${{steps.slack-post-message.outputs.ts}}
```

Comment:
Oof. That took a lot more iteration than I care to admit, but this was the bit I was missing: to set your custom action's output you have to define where it should pull the value from. 🤦

Reply:
😅